### DISCUSSION TITLE
测试

### DISCUSSION THREAD

**Discussion by Arataki-Amerike** [share121/inter-knot#4837] [灌水]
测试一下图片上传～
![1726067178](https://github.com/user-attachments/assets/80e34d7c-9c4c-49d0-b46a-4583063165b6)

**Comment by Arataki-Amerike** [comment]
成功了，好诶
![Screenshot_20230602_021452_com huawei systemmanager](https://github.com/user-attachments/assets/b3c5076c-ee9e-498e-b386-5e7eb0c2b260)

**Comment by Junjun280** [comment]
恭喜100天达成！！！<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >
